.github/workflows/pyinstaller-windows.yml

```diff
@@ -0,0 +1,34 @@
+name: Package Application with Pyinstaller
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+
+    - name: Install pyinstaller
+      run: python -m pip install pyinstaller -r mathlibtools/pyinstaller-requirements.txt
+
+    - name: Run pyinstaller
+      run: python -m PyInstaller mathlibtools/leanproject.spec
+
+    - name: Check the resulting exe functions
+      run: ${{ github.workspace }}/dist/leanproject.exe --help
```

Comment:
I'm quite worried the exe only works if in this specific location. Have we tested that it works when downloaded (on a system with no installation of mathlib-tools)?

Reply:
I have not which is what I meant by "someone with a windows machine should try downloading that".

Definitely agree it's worth doing before merging, I have the same sneaky suspicion given how magical pyinstaller is.

Reply:
Yes I tested that the download works on a system with no installation ever of mathlib-tools.

Reply:
Where does this stand? Is there any obstacle to accepting this pull request?

Reply:
I'll try to remember to test this tomorrow. The config looks suspect to me still, but that's only a first impression having never used pyinstaller before.

Reply:
I think you are remembering that the .spec file included a pathname on my PC which I fixed on May 4. https://github.com/leanprover-community/mathlib-tools/pull/101/commits/6f75794175093f5ddcf0b8b9ac637fd2e2e48659 That was just a relic of how the file was generated.

Reply:
I tried it out, and after trying a few argument variants, windows defender tried to quarantine it as malware. Unless pyinstaller itself is compromised I assume it's a false alarm.

Reply:
Yes, that will happen to the first so many people who download it. I have been involved with a distributed computing project (http://vdwnumbers.org/) for ten years--this happens for a few days every time we upgrade the exe files, until enough people click on it as OK. Chrome says "leanproject.exe.zip is not commonly downloaded and may be dangerous.

Reply:
What is the status of this pull request?